homeassistant/components/shelly/binary_sensor.py

```diff
@@ -15,8 +16,18 @@
 )
 
 SENSORS = {
-    ("device", "overtemp"): BlockAttributeDescription(name="overtemp"),
-    ("relay", "overpower"): BlockAttributeDescription(name="overpower"),
+    ("device", "overtemp"): BlockAttributeDescription(
+        name="overtemp", device_class=DEVICE_CLASS_PROBLEM
```

Comment:
These names should be human readable names.

Reply:
I changed it to "Over Temperature" but I wonder if "Overheating" will be better.

Reply:
I like Overheating

Reply:
@bieniu what do you think about unifying the names? Right now we have `Overheating` and `Over Power`. Both should be written in the same way.

Reply:
What's your suggestion?

Reply:
When I typed into google `over heating sensor` it is suggesting `overheating sensor` same with `over power sensor`.
So I think all would be written without space in the name, so: `Overheating` and `Overpower`

Reply:
I'm not a native speaker and my English is rather poor so others should comment on this. You can always add PR.

Reply:
@bieniu I think your a better speaker than I am 🙂
Will do, we will discuss that change there.